warehouse/templates/packaging/detail.html

```diff
@@ -164,6 +164,7 @@ <h3 class="sidebar-section__title">GitHub statistics</h3>
             <strong>Open issues/PRs: </strong>
             <span class="github-repo-info__item" data-key="open_issues_count"></span>
           </a>
+          <p>View additional statistics for this project via <a href="https://libraries.io/pypi/{{ release.project.name }}">Libraries.io</a>, or by using <a href="https://packaging.python.org/guides/analyzing-pypi-package-downloads/">Google BigQuery</a></p>
```

Comment:
This line will not be shown if the release doesn't have a `github_repo_info_url`. We'll need to add a fallback to the conditional which still renders the original section.